modules/core/src/main/scala/lucuma/odb/api/model/targetModel/SourceProfileModel.scala

```diff
@@ -303,10 +303,14 @@ object SourceProfileModel {
 
     import lucuma.odb.api.model.targetModel.SourceProfileModel.BandNormalizedInput._
 
-    override val create: ValidatedInput[BandNormalized[T]] =
-      (sed.notMissingAndThen("sed")(_.toUnnormalizedSed),
-       brightnesses.notMissingAndThen("brightnesses")(brightnessCreator[T])
-      ).mapN { (sed, bright) => BandNormalized(sed, bright) }
+    override val create: ValidatedInput[BandNormalized[T]] = {
+      // we can't update lucuma-core thus we need to default to somthing
+      val modelSed = sed.toOption.map(_.toUnnormalizedSed)
+        .getOrElse(UnnormalizedSED.StellarLibrary(StellarLibrarySpectrum.O5V).validNec)
+      (modelSed, brightnesses.notMissingAndThen("brightnesses")(brightnessCreator[T])).mapN { (sed, bright) =>
+        BandNormalized(sed, bright)
+      }
+    }
```

Comment:
In the `edit` below, we should probably switch to `sed.validateNullable` and then I guess treat an attempt to assign `null` as `05V`?

Maybe

```
s <- sed.validateNullable(_.toUnnormalizedSed).liftState
_ <- BandNormalized.set[T] := s.map(_.getOrElse(UnnormalizedSED.StellarLibrary(StellarLibrarySpectrum.O5V)))
```

Reply:
Right, at the moment explore won't try to send a null but maybe we should support it for api clients?

Reply:
Oh, well especially in that case it doesn't seem like it's worth the trouble considering tmp-api is EOL.